### DISCUSSION TITLE
Where is the sqlite3@4.2.0 installed?

### DISCUSSION THREAD

**Discussion by regunakyle** [mtxr/vscode-sqltools#869] [Q&A] [answered]
Title. 
I can't find it in my global npm installation and the VSCode extension folder.
So where is the sqlite3 module installed?

**Comment by gjsjohnmurray** [comment, marked as answer]
- macOS: ~/Library/Application Support/vscode-sqltools
- Windows: %LOCALAPPDATA%\vscode-sqltools\Data (for example, C:\Users\USERNAME\AppData\Local\vscode-sqltools\Data)
- Linux: ~/.local/share/vscode-sqltools (or $XDG_DATA_HOME/vscode-sqltools)

These paths are obtained by calling `paths.data` in the [env-paths](https://www.npmjs.com/package/env-paths) package.
